resources/startup.sh

```diff
@@ -8,33 +8,33 @@ function mask2cidr() {
   IFS=.
```

Comment:
I know you didn't change line. Anyhow this line is **really** a gotcha and we have to take care of it.

`$IFS` is an exported, systemwide variable. If we change this other scripts may misbehave or act weirdly in silence. It is best to save $IFS to a backup var and restore it afterwards we are finished with splitting by dots.

```
  IFS_original="$IFS"
  IFS=.
  
  # here goes the cidr number crunching

  IFS="$IFS_original"
```

Reply:
Good catch!
See also https://github.com/cloudogu/sonar/blob/df544b84c4beb4fdfbc20ebbf7fa3cb9deb919bf/resources/startup.sh#L350